game/hud/src/widgets/Compass/index.scss

```diff
@@ -1,32 +1,25 @@
 /**
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-@font-face {
-  font-family: TitilliumBold;
-  src: url(../fonts/TitilliumWeb-Bold.ttf); 
-}
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
 
+$cu-styles-root: '../../../interface-lib/camelot-unchained/styles/';
+@import "camelot-unchained/lib/index.scss";
```

Comment:
I don't see any uses of the old cu styles in the compass at all... remove the reference to it.